mu4e/mu4e-compose.el

```diff
@@ -461,7 +461,7 @@ The message is resent as-is, without any editing. See
           "Resend message to address: " mu4e--contacts-set)))
   (let ((msg (mu4e-message-at-point)))
     (with-temp-buffer
-      (mu4e--prepare-draft msg)
+      (mu4e--prepare-draft 'new msg) ;; Resend as fake new?
```

Comment:
@djcb This is the point I’m the less sure about. Do not hesitate to comment if this is completely wrong.